transport/src/main/java/io/netty/channel/embedded/EmbeddedEventLoop.java

```diff
@@ -51,6 +51,26 @@ void runTasks() {
         }
     }
 
+    long runScheduledTasks() {
+        long time = AbstractScheduledEventExecutor.nanoTime();
+        for (;;) {
+            Runnable task = pollScheduledTask(time);
+            if (task == null) {
+                return nextScheduledTaskNano();
+            }
+
+            task.run();
+        }
+    }
+
+    long nextScheduledTask() {
+        return nextScheduledTaskNano();
+    }
+
+    void cancelScheduledTasks() {
+        cancelDelayedTasks();
```

Comment:
After renaming `cancelDelayedTasks()` to `cancelScheduledTasks()`, you can just call `super.cancelScheduledTasks()` rather than introducing another method.